src/Pos/Genesis.hs

```diff
@@ -185,7 +185,9 @@ genesisUtxo :: StakeDistribution -> Utxo
 genesisUtxo sd =
     M.fromList . zipWith zipF (stakeDistribution sd) $ genesisAddresses
   where
-    zipF coin addr = (TxIn (unsafeHash addr) 0, (TxOut addr coin, []))
+    zipF coin addr =
+        ( TxIn (unsafeHash addr) 0
+        , (TxOutAux {toaOut = TxOut addr coin, toaDistr = []}))
```

Comment:
Maybe in this particular case it's obvious, but code usually tends to become non-obvious sooner or later, so it's usually good to use explicit field names in advance unless it makes code much worse (in this case it doesn't I suppose).